packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -392,11 +392,7 @@ export function TransferPanel() {
       const switchTargetChainId = latestNetworks.current.sourceChain.id
       try {
         await switchNetworkAsync?.(switchTargetChainId)
-      } catch (e) {
-        if (isUserRejectedError(e)) {
-          return
-        }
-      }
+      } catch (e) {}
```

Comment:
What's the purpose of an empty catch block here now? I think we should remove it or log the error to Sentry?